source/common/io/BUILD

```diff
@@ -11,7 +11,7 @@ envoy_package()
 envoy_cc_library(
     name = "io_uring_impl_lib",
     srcs = select({
-        "@envoy//bazel:linux": ["io_uring_impl.cc"],
```

Comment:
im curious why these werent remove previously - iiuc the formatter would do that